src/test/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/GovernmentPublicationMapperTest.java

```diff
@@ -0,0 +1,75 @@
+package org.folio.marc4ld.service.marc2ld.mapper.mapper;
+
+import static org.apache.commons.lang3.StringUtils.repeat;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.ld.dictionary.PropertyDictionary.LINK;
+import static org.folio.ld.dictionary.PropertyDictionary.TERM;
+import static org.folio.marc4ld.mapper.test.TestUtil.OBJECT_MAPPER;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.dto.MarcData;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.MarcFactory;
+
+class GovernmentPublicationMapperTest {
+
+  private static final String LINK_PREFIX = "http://id.loc.gov/vocabulary/mgovtpubtype/";
+  private final GovernmentPublicationMapper mapper = new GovernmentPublicationMapper(OBJECT_MAPPER);
+  private final MarcFactory factory = MarcFactory.newInstance();
+
+  @ParameterizedTest
+  @CsvSource(value = {
+    "a , a, Autonomous",
+    "c , c, Multilocal",
+    "f , f, Federal",
+    "i , i, International Intergovernmental",
+    "l , l, Local",
+    "m , m, Multistate",
+    "o , g, Government",
+    "s , s, State"
+  })
+  void shouldMapGovernmentPublication(String charInMarc, String expectedLinkSuffix, String expectedTerm) {
+    // given
+    Resource resource = new Resource().setDoc(OBJECT_MAPPER.convertValue(Map.of(), JsonNode.class));
```

Comment:
var (same below and in TargetAudienceMapperTest)

Reply:
Done